### PR TITLE
fix: multiple identity support

### PR DESCRIPTION
**Pull request by SunriseFox**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
         "node": true
     },
     "rules": {
+        "no-debugger": "error",
```
Comment (Jack-Works) [inline .eslintrc.json]: 🤔

File: src/extension/background-script/CryptoServices/decryptFrom.ts [hunk 2]
```diff
@@ -138,7 +138,7 @@ export async function* decryptFromMessageWithProgress(
         }
 
         try {
-            if (by.equals(whoAmI)) {
+            try {
```
[the remark this answers is not in the document]
Comment (SunriseFox) [reply src/extension/background-script/CryptoServices/decryptFrom.ts]: move this try block outside is ok... but put it inside... seems also ok

File: .eslintrc.json (hunk 1 above)
Comment (SunriseFox) [reply .eslintrc.json]: in case I commit with `debugger` statement again

File: src/components/shared-settings/settings.ts [hunk 3]
```diff
@@ -18,3 +21,28 @@ export const disableOpenNewTabInBackgroundSettings = createNewSettings<boolean>(
             "Many of Maskbook features relies on this behavior. Disable this behavior will limit Maskbook's functionality",
     },
 )
+
+const createProxiedSettings = (key: string) => {
+    const target: { [key: string]: ValueRef<string> | string } = {}
```
Comment (Jack-Works) [inline src/components/shared-settings/settings.ts]: Should this `target` defined in the global scope?

Comment (SunriseFox) [reply src/components/shared-settings/settings.ts]: no. it is only for current settings.

**Pull request merged**